server/src/controllers/confirm.user.controllers/confirm.token.controller.ts

```diff
@@ -12,12 +12,20 @@ export const confirm = async (req: Request, res: Response, next: NextFunction):
             throw new Error(`Token doesn't exist`);
         }
 
-        await User.findOneAndUpdate(
+        const userExist = await User.findOneAndUpdate(
```

Comment:
try to move all request to backend in separate method in directory dbQuery for example

you can create file userQueries.tsx in order to connect to User Scheme. In this file can be methods emailExist, userExist, findOneAndUpdate etc.

Then any team member could reuse it and don't repeat the same code.